### DISCUSSION TITLE
CuratorPicker does not display properly after clicking.

### DISCUSSION THREAD

**Discussion by 000yuying** [awcodes/filament-curator#319] [Q&A]
I'm facing an issue where the CuratorPicker is not displaying when I click it.
It was working fine in my local testing, but this issue only appeared after going live.

In developer mode, it shows these messages. 
How can I troubleshoot this?

![截圖 2023-10-05 16 15 39](https://github.com/awcodes/filament-curator/assets/109067769/cc34dd28-d7ca-421b-8b8e-631cd47c6818)

**Comment by awcodes** [comment]
View cache on production maybe. If everything works locally, I'm not sure there's much I can do about issues you're seeing on your production server.
